### DISCUSSION TITLE
Change status colors for running vs successful query in sqllab

### DISCUSSION THREAD

**Discussion by cooley-pe** [apache/superset#18402] [Ideas]
**Is your feature request related to a problem? Please describe.**
I have a lot of trouble distinguishing between the shades of green signifying a running vs successful query in sqllab. 

**Describe the solution you'd like**
Ideally these would be two different colors for running vs successful. Maybe yellow and green respectively?

**Describe alternatives you've considered**
If there isn't an entirely different color that makes sense, making the greens more distinct would be helpful.

**Additional context**
Success on the left and running on the right:
![Screen Shot 2020-09-30 at 3 40 02 PM](https://user-images.githubusercontent.com/64038793/94747318-6523ea00-0333-11eb-97d5-fb4f398dd39a.png)

[a comment by an automated account is not shown]

**Comment by sahilkamboj3** [comment]
Hey @junlincc , I'd like to contribute. Is the issue available to work on?

**Comment by junlincc** [comment]
@sahilkamboj3 Hi! sorry I missed your comment 🙏 proposed solution sounds reasonable. Please go ahead and open a PR, we will review for sure. 

cc @steejay

**Comment by Steejay** [comment]
Thanks for raising this issue @cooley-pe! 

@sahilkamboj3 Solution sounds good but lmk if/when you need further design input

**Comment by yousoph** [comment]
A suggestion on #13648 was to have a hollow green o for the running icon and a filled in o for the successful query.

**Comment by jamesfurnary** [comment]
Hi @Steejay and @sahilkamboj3, do you have an update on this? CC @cooley-pe 

Thanks,
James

**Comment by justinpark** [comment]
I created a PR for this change: https://github.com/apache/superset/pull/22050
